server/tests/planning_export/planning_export_tests.py

```diff
@@ -896,15 +896,18 @@ def test_bilingual_planning_advisory_export_tomorrow(self):
             )
             self.app.data.insert("desks", test_desks)
 
-            event_1_id = ObjectId()
-            event_2_id = ObjectId()
+            # Planning items
+            planning_1_id = ObjectId()
+            planning_2_id = ObjectId()
+            planning_3_id = ObjectId()
 
-            bilingual_events = [
+            planning_items = [
                 {
-                    "_id": event_1_id,
+                    "_id": planning_1_id,
+                    "type": "planning",
+                    "slugline": "planning-dutch-text",
                     "name": "Test Event NL",
-                    "slugline": "test-event-nl",
-                    "definition_long": "Dutch description of the event",
+                    "description_text": "Dutch description of the event",
                     "dates": {
```

Comment:
I suppose it's not used, but it wouldn't be in planning items so better remove it so it's not confusing

Reply:
Thanks Petr, actually description_text is used in planning items

Reply:
I mean `dates`